File: Go/File/file-exists.go
package main

import (
	"fmt"
	"os"
)

func main() {
	// Check if the file exists.
	fmt.Println(fileExists("assets/valid/valid-txt.txt"))
}

/* It checks if the file exists
If the file exists, it returns true
If the file does not exist, it returns false */
func fileExists(filename string) bool {
	info, err := os.Stat(filename)
	if err != nil {
		return false
	}
	return !info.IsDir()
}

File: Dart/File/file_exists.dart
import "dart:core";
import "dart:io";

void main() {
  // Check if a file exists.
  print(fileExists("assets/valid/j3U5uJY779L49q98MX86iFsxs2kY9ew3/README.md"));
}

// Check if a file exists and return a boolean
bool fileExists(final String path) {
  return File(path).existsSync();
}
